templates/org/aac/aac-browser-header.yml

```diff
@@ -14,8 +14,8 @@ azureCategories:
   - <There can be more than one category>
 products:
   - <Choose 1-5 products from the list at https://review.learn.microsoft.com/help/contribute/architecture-center/aac-browser-authoring#products>
-  - <1-5 products--no more than 5!>
-name: <The H1 title is a phrase with a present tense verb that describes the scenario (no gerunds, "-ing" verbs). Example: "Use Azure monitoring to integrate security components">
+  - <Include between 1 and 5 products, but no more than five>
+name: <The H1 title is a phrase with a present tense verb that describes the scenario. Don't use gerunds, or "-ing" verbs. For example: "Use Azure monitoring to integrate security components">
 summary: <Write a summary. Can be same as description.>
 thumbnailUrl: /azure/architecture/browse/thumbs/<browse-png-filename>.png
 content: |
```

Comment:
On line 22, please make `include` read `INCLUDE`.  Sorry, github won't let me put this comment on line 22.